src/bin/vip-sync.js

```diff
@@ -22,182 +22,187 @@ command( {
 	childEnvContext: true,
 	module: 'sync',
 	requireConfirm: 'Are you sure you want to sync from production?',
-} ).argv( process.argv, async ( arg, opts ) => {
-	const api = API();
-	let syncing = false;
-
-	await trackEvent( 'sync_command_execute' );
-
-	try {
-		await api.mutate( {
-			mutation: gql`
-				mutation SyncEnvironmentMutation($input: AppEnvironmentSyncInput) {
-					syncEnvironment(input: $input) {
-						environment {
-							id
+} )
+	.example(
+		'vip @example-app.develop sync',
+		'Sync the production environment database of the "example-app" application to the development environment.'
```

Comment:
Should this be "to the develop environment" since develop is the name of the environment used in the command?

Reply:
Indeed! Thank you.